packages/api-v2/src/hooks/__tests__/useAvailableCTraderAccounts.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useQuery from '../../useQuery';
+import useAvailableCTraderAccounts from '../useAvailableCTraderAccounts';
+
+jest.mock('../../useQuery');
+
+describe('useAvailableCTraderAccounts', () => {
+    const mockUseQuery = useQuery as jest.Mock;
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should return modified accounts when data is available', () => {
```

Comment:
hey hey @hasan-deriv 
we are trying to follow new rules to describe test cases (in particular we are trying to avoid **should**):
When writing tests using Jest or other testing frameworks, it's a common practice to follow certain conventions for naming and describing your test cases. One such convention — adopted by many developers and teams — is avoiding the use of "should" in test descriptions. This idea stems from the thought that there's a more expressive and clear way to describe test cases without using "should".
e.g.: **returns** modified accounts when data is available
passes through other fields from useAuthorizedQuery
etc.